### ISSUE TITLE
Excessive deviation in the estimated size of the checkerboard grid

### ISSUE THREAD

**Issue by ThanksForYourCreating** [acfr/cam_lidar_calibration#62]
#Actual size of my checkerboard
chessboard:
  pattern_size:
    height: 5
    width: 3
  square_length: 45
  board_dimension:
    width: 301
    height: 400
  translation_error:
    x: 0
    y: 0
    BUT!!!
When I try to get a test with the V2 version, the terminal outputs an outrageous deviation in the size of the checkerboard grid boards.
   --- Sample 15 ---

Measured board has: dimensions = 4029.22653 x 20543.62131 mm; area = 0.00000m^2

Distance =  1.91 m

Board angles     =   nan, 1.47,118.06,48.83 degrees

Board area       = 89.36395 m^2 (+89.36395 m^2)

Board avg height = 20543.62mm (+20543.62mm)

Board avg width  = 4029.23mm (+4029.23mm)

Board dim        = 1746.05,17657.71,23429.54,6312.40 mm

Board dim error  = 49145.70

Avg Lidar Normal =  -0.97,  -0.11,  -0.05 

Avg Lidar Centre = 1368.31,  34.93, 1336.80 

Std Dev Lidar Normal =   0.30,   0.05,   0.05 

Std Dev Lidar Centre = 867.29, 166.60, 13839.96 

[ERROR] [1720871945.821198015]: Plane fitting error, LiDAR board dimensions incorrect; discarding sample - try capturing again
As you can see, the dimensions of the checkerboard grid were misclassified as 4029.22653 x 20543.62131 mm
Even if I wanted to adjust the
aram name="distance_offset_mm" 
I tried value="-100" / -800 / -1000 /-1400/-2000/-10000/-14000
But I found that when value is small it still can't correct board grids that deviate too much, but when value is greater than 500 children less than 1400, the Experimental Region's corresponding board grid point cloud squeezes together, resulting in it not being recognized correctly, either prompting that there are not enough lines on the board even though it looks to the human eye like it's more than 7 lines, or prompting a second try at other poses.
In short I can't get the checkerboard board dimensions close to the true value by adjusting distance_offset_mm.

**Comment by zhangyuran-gg** [comment]
你好，我有同样的问题。你解决这个问题了吗

**Comment by ghost** [comment]
I encountered the same problem as you. Have you solved it?
